user-console/src/main/java/org/pentaho/mantle/client/commands/NewDropdownCommand.java

```diff
@@ -195,6 +213,9 @@ public void onClick( ClickEvent event ) {
             popup.setPopupPosition( anchorWidget.getAbsoluteLeft(), anchorWidget.getAbsoluteTop()
                 + anchorWidget.getOffsetHeight() );
             popup.show();
+            if ( buttonPanel.getWidgetCount() > 0 ) {
+              buttonPanel.getWidget( 0 ).getElement().focus();
```

Comment:
( (Button) buttonPanel.getWidget( 0 ) ).setFocus( true );

Reply:
Thanks for addressing this.
Unfortunately, I have realized that there is another possible issue.

There is a special condition in which the child widgets may be an HTML widget. Search for:
```java
buttonPanel.add( new HTML( "<hr style='color: #a7a7a7' />" ) );
```

This would make the above proposed cast to fail. Your initial code would not fail with the cast error, but would fail to set the focus (because an `hr` is not focusable).

The following proposal should solve this issue, once and for all.
Replace lines 216-218 by:
```java
Focusable firstButton = ElementUtils.findFirstKeyboardFocusableDescendant( popup );
if ( firstButton != null ) {
  firstButton.setFocus( true );
}
```